Readme.md

```diff
@@ -36,6 +36,7 @@ Add the location for `rcop` to your `$PATH` and the `commit-msg` from this repos
 Here are the command line options you can use with rcop:
 
 `--dont-exit-on-errors` or `-e`: By default, the command line utility exits with a non-zero exit code when it encounters an error. If you pass this option, it will print the error message and continue running.
+`--allow-caps-types` or `-c`: Allow all defined commit types to be uppercase as well as lowercase (e.g., "feat" and "FEAT"). 
```

Comment:
`--ignore-case`, `-i` is probably a better name as it's a common flag (see ie grep)